docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -44,25 +44,25 @@ A conformant client library MUST make a GET request with the headers:
     * iOS MUST be `expo-platform: ios`.
     * Android MUST be `expo-platform: android`.
     * If it is not one of these platforms, the server SHOULD return a 400 or a 404
-2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file. 
+2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file.
 3. Any headers stipulated by a previous responses' [server defined headers](#manifest-response-headers).
 
-A conformant client library MUST also send at least one of `accept: application/expo+json` or `accept: application/json`, though SHOULD send `accept: application/expo+json, application/json` with the order following the preference ordering for the accept header specified in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.3.2).
+A conformant client library MUST also send at least one of `accept: application/expo+json`, `accept: application/json`, or `accept: multipart/mixed` though SHOULD send `accept: application/expo+json, application/json, multipart/mixed` with preferences expressed with "q" parameters as specified in [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.1).
 
 Example:
 ```
-accept: application/expo+json, application/json
+accept: application/expo+json;q=0.9, application/json;q=0.8, multipart/mixed
```

Comment:
Might be clearer to put multipart/mixed first or add a q value. q=1.0 is the default if unspecified. Or just leave off all the q values since they don't matter here.

Reply:
I actually think this example is useful to show that:
a) order doesn't matter
b) it defaults to 1.0

i.e. it's useful to show that the other spec is being followed.

Reply:
OK, in that case I think it is useful to say in plain English what the preference ordering is.